src/e2e-test/resources/pluginParameters.properties

```diff
@@ -108,6 +112,10 @@ pubsubDelimiter=@
 
 ## GCSDELETE-PLUGIN-PROPERTIES-START
 gcsDeleteObjectsList=testdata/GCS_RECURSIVE_TEST/recursiveFile1.csv,testdata/GCS_RECURSIVE_TEST/recursiveFile2.csv
+gcsDeleteObjectsList2=testdata/GCS_WILDCARD_TEST/wildcardFile2.csv,testdata/GCS_WILDCARD_TEST/wildcardFile3.csv
+gcsDeleteObjectsList3=testdata/GCS_WILDCARD_TEST/test
```

Comment:
`gcsWildcardPath3=testdata/GCS_WILDCARD_TEST/text*`
You have `text` above but `test` here. Is it a type?

Reply:
oh yeah!